res/css/views/elements/_GenericEventListSummary.scss

```diff
@@ -43,9 +111,6 @@ limitations under the License.
 .mx_GenericEventListSummary_toggle {
     color: $accent;
     cursor: pointer;
-    float: right;
-    margin-right: 10px;
-    margin-top: 8px;
```

Comment:
Moved to https://github.com/matrix-org/matrix-react-sdk/pull/8713/files#diff-f5e6e48067f0db9e4d21def8a6998dc916f40442fe6618996b2484867dee705dR23